docs/Containers.rst

```diff
@@ -331,28 +316,63 @@ Example of a generated container recipe
 
 Below is an example of container recipe for that was generated by EasyBuild, using the following command::
 
-    eb Python-3.6.4-foss-2018a.eb OpenMPI-2.1.2-GCC-6.4.0-2.28.eb -C --container-base-image shub:shahzebsiddiqui/eb-singularity:centos-7.4.1708 --experimental
-
-It uses the ``shahzebsiddiqui/eb-singularity:centos-7.4.1708`` base container image that is available from Singularity hub
-(see https://singularity-hub.org/collections/143).
+    eb Python-3.6.4-foss-2018a.eb -C --container-config bootstrap=yum,osversion=7 --experimental
 
 .. code::
 
-    Bootstrap: shub
-    From: shahzebsiddiqui/eb-singularity:centos-7.4.1708
+    Bootstrap: yum
+    OSVersion: 7
+    MirrorURL: http://mirror.centos.org/centos-%{OSVERSION}/%{OSVERSION}/os/x86_64/
+    Include: yum
 
     %post
-    yum --skip-broken -y install openssl-devel libssl-dev libopenssl-devel
-    yum --skip-broken -y install libibverbs-dev libibverbs-devel rdma-core-devel
-
-
-    # upgrade easybuild package automatically to latest version
-    pip install -U easybuild
+    yum install --quiet --assumeyes epel-release
+    yum install --quiet --assumeyes python setuptools Lmod
+    yum install --quiet --assumeyes bzip2 gzip tar unzip xz
+    yum install --quiet --assumeyes curl wget
+    yum install --quiet --assumeyes patch make
+    yum install --quiet --assumeyes file git which
+    yum install --quiet --assumeyes python-pip
+    yum install --quiet --assumeyes gcc-c++
```

Comment:
why are this all separate calls to yum? It will be quicker when combined.

Reply:
That's just because they're listed separately in the code, I doubt it'll make a huge difference when we do everything in one go though?

Reply:
Just timed it, total time (incl. `pip install easybuild` and `eb bzip2-1.0.6.eb`), I don't see much (if any) difference between running multiple `yum install` commands vs one big `yum install`.

In addition, the `yum install epel-release` has to be done separately, otherwise installing `python-pip` and `Lmod` fails.

I don't think it's worth making the generated recipe harder to read for humans for a matter of seconds at best?